### DISCUSSION TITLE
Architecture Diagram

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#725] [General]
I created a diagram to visualize the client/server architecture and dependencies... hopefully folks find it useful...

https://github.com/oqtane/oqtane.framework#architecture

**Comment by kevind2000** [comment]
Wonderful! 

Can you please also share Oqtane Core Entity ERD? The one I saw in your blog is kind of fuzzy - I mean the image not the diagram.

**Reply by kevind2000** [reply]
Never mind. Your time is better spent on something else. I can create ERD myself easily.

**Comment by ncorovic1** [comment]
I created an ERD diagram via SSMS according to the current entity definition. It might be helpful for future starters.

![Oqtane ERD](https://user-images.githubusercontent.com/11291134/138077043-ccdec23c-d1c5-4d39-9b3f-e038c3aa7401.jpg)
